yt_dlp/extractor/thirtydaysinger.py

```diff
@@ -0,0 +1,102 @@
+from .wistia import WistiaIE
+from ..utils import (
+    clean_html,
+    get_elements_html_by_class
+)
+
+
+class ThirtyDaySingerBase(WistiaIE):
+    def _extract_for_url(self, url):
+        lesson_index = self._match_id(url)
+        webpage = self._download_webpage(url, lesson_index)
+        match = next(self._extract_wistia_async_embed(webpage))
+        embed_config = self._download_embed_config('medias', match.group('id'), url)
+
+        embed_infojson = self._extract_media(embed_config)
+        webpage_infojson = self._extract_webpage_data(webpage)
+
+        return {**embed_infojson, **webpage_infojson}
+
+    def _extract_webpage_data(self, webpage):
+        title = self._html_search_regex(r'<h1>([^<]+)</h1>', webpage, 'title')
+        fallback_title = self._html_extract_title(webpage)
+        description = self._html_search_meta('description', webpage, fatal=False)
+
+        return {
+            'title': title or fallback_title,
+            'description': clean_html(self._format_html_list(description))
+        }
+
+    # The site makes extensive use of HTML lists for formatting and `clean_html`
+    # doesn't handle them well. This is needed to keep lists readable.
+    def _format_html_list(self, html):
+        replacements = {
+            '<ul>': '<br>',
+            '</ul': '<br>',
+            '<li>': '<br>- ',
+            '</li>': ''
+        }
+
+        for k, v in replacements.items():
+            html = html.replace(k, v)
+
+        return html
+
+
+class ThirtyDaySingerIE(ThirtyDaySingerBase):
+    _VALID_URL = r'(https?://)?www.30daysinger.com/tutorial/[\w-]+/(?P<id>[\w-]+)'
+
+    _TESTS = [{
+        'url': 'https://www.30daysinger.com/tutorial/30-day-beginner-course-with-jonathan-estabrooks/1',
+        'md5': '56bb11529b9777899b27b599d4b16cf6',
+        'info_dict': {
+            'id': 'tegd38l3d5',
+            'ext': 'mp4',
+            'thumbnail': 'http://embed.wistia.com/deliveries/c26a85cb98e32efa8a5e12a0576e63355af66230.jpg',
+            'upload_date': '20190608',
+            'description': 'md5:d3291de8988be57b1d3e411126ba4d33',
+            'duration': 344.22,
+            'timestamp': 1559952526,
+            'title': 'Welcome to 30 Day Singer'
+        }
+    }]
+
+    def _real_extract(self, url):
+        return self._extract_for_url(url)
+
+
+class ThirtyDaySingerPlaylistIE(ThirtyDaySingerBase):
+    _URI_BASE = 'https://www.30daysinger.com'
+    _VALID_URL = r'(https?://)?www.30daysinger.com/tutorial/(?P<id>[\w-]+)'
```

Comment:
so it doesn't match video URLs:
```suggestion
    _VALID_URL = r'(https?://)?www.30daysinger.com/tutorial/(?P<id>[\w-]+)/?(?:$|[?#])'
```